src/components/20-molecules/datepicker/e2e.js

```diff
@@ -36,5 +36,55 @@ test.describe('datepicker', () => {
         '29.02.2020 (of 1 submittable elements)'
       );
     });
+
+    test('should just add years of a given range', async ({ page }) => {
+      await page.goto(fixtureURL('components-datepicker--datepicker'));
+
+      const datepicker = await page.locator('axa-datepicker');
```

Comment:
It would arguably more stable to use the selector `[data-test-id="datepicker"]`, since that is explicitly provided for testing.